config/wpcalypso.json

```diff
@@ -49,6 +49,7 @@
 		"manage/export/guided-transfer": true,
 		"manage/import/medium": true,
 		"manage/jetpack": true,
+		"manage/media": true,
```

Comment:
For anyone reading this would turn on the section for everything but Deskftop, Prod and Stage